tmt-app/src/Components/MajorPage.jsx

```diff
@@ -13,11 +13,19 @@ function MajorPage() {
 
   searchData(params.major);
   return (
-    <Container>
+    <Container sx={{ minWidth: "70%" }}>
+      <Breadcrumb
+        previousPage={{
+          category: "Major",
+          major: data.name,
+          parent: "Home",
+          path: "",
+        }}
```

Comment:
Is it possible to bypass the two steps from your previousPage prop to the breadcrumbs variable in the component and directly iterate off the prop? Could save you a few steps and provide additional flexibility for future use